Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -414,13 +414,32 @@ extension SourceFilesList {
 
 extension ResourceFileElements {
     fileprivate static func from(resources: [PackageInfo.Target.Resource], path: AbsolutePath) -> Self? {
-        let resourcesPaths = resources.map { path.appending(RelativePath($0.path)) }
+        var resourcesPaths = resources.map { path.appending(RelativePath($0.path)) }
+        resourcesPaths.append(contentsOf: defaultResources(from: path))
         guard !resourcesPaths.isEmpty else { return nil }
         return .init(resources: resourcesPaths.map { absolutePath in
             let absolutePathGlob = absolutePath.extension != nil ? absolutePath : absolutePath.appending(component: "**")
             return .glob(pattern: Path(absolutePathGlob.pathString))
         })
     }
+
+    private static let defaultResourceFileExtensions = ["xib", "storyboard", "xcdatamodeld", "xcmappingmodel", "xcassets", "lproj"]
+
+    private static func defaultResources(from path: AbsolutePath) -> [AbsolutePath] {
+        let files = FileHandler.shared.filesAndDirectoriesContained(in: path) ?? []
```

Comment:
Are you sure SPM searches the whole package, and not some specific folder (e.g. sources are searched in `Sources/Target` if no parameter is specified.
Also, can you add:
- a unit test in PackageInfoMapperTests
- the dependencies that caused problems to the `app_with_spm_dependencies` fixture

Reply:
From what I see [here](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#resource) looks like the logic of searching in the `path` is correct

Reply:
The `app_with_spm_dependencies` fixture is failing, you can try to generate it locally and build it to see what the problem is